docs/compatibility.md

```diff
@@ -2,262 +2,161 @@ List of supported features, depending on the Docker Compose version.
+|     - delay                  |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - failure_action         |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - max_failure_ratio      |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - monitor                |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - order                  |  SK |  SK |  SK |  SK |     |     |     |     |     |     |     |     |     |
+|     - parallelism            |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+| - devices                    |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - depends_on                 |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+| - dns                        |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+| - dns_search                 |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+| - domainname                 |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - entrypoint                 |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - env_file                   |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+| - environment                |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - expose                     |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+| - external_links             |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - extra_hosts                |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - healthcheck                |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |     |
+|   - disable                  |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+|   - interval                 |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+|   - retries                  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+|   - start_period             |  CS |  CS |  CS |  CS |     |     |     |     |
+|   - test                     |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+|   - timeout                  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - hostname                   |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - image                      |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - init                       |  *  |     |     |     |     |     |     |     |     |     |     |     |     |
```

Comment:
Probably not support in Kubernetes if it's in 3.7, might be wrong though

Reply:
Fixed.